content/blog/2025/09/esp-wifi-remote/index.md

```diff
@@ -0,0 +1,189 @@
+---
+title: "*esp-wifi-remote*: The remote control to your Wi-Fi"
+date: 2025-09-05
+showAuthor: false
+summary: "**esp-wifi-remote** provides traditional `esp_wifi` functionality on non-WiFi targets, allowing them to act as a `host` that communicates with a WiFi-capable `slave` device through a transport layer."
+authors:
+  - david-cermak
+tags: [  "ESP-IDF", "Wi-Fi", "esp-hosted", "ESP32"]
```

Comment:
I suggest change the title and summary to concentrate around `esp_wifi_remote` main point that is to provide wifi connectivity transparently. The information is there but feels to be a convoluted way to express that.